app/src/util/stringUtil.ts

```diff
@@ -0,0 +1,6 @@
+
+export const isEmailValid = (email: string): boolean => {
```

Comment:
not sure if `stringUtil` is the right place for this 😂  Didn't create an `emailUtil` cos not sure what else would be put in there. maybe we can move this to an emailUtil when we have other email things ?